frontend/event/eventController.js

```diff
@@ -404,9 +404,9 @@
         }
 
         function loadEventDates() {
-            dialogCtrl.start_time = new Date(dialogCtrl.dateChangeEvent.start_time);
-            dialogCtrl.event.start_time = new Date(dialogCtrl.dateChangeEvent.start_time);
-            dialogCtrl.event.end_time = new Date(dialogCtrl.dateChangeEvent.end_time);
+            dialogCtrl.start_time = moment.utc(dialogCtrl.dateChangeEvent.start_time)._d;
```

Comment:
why the property _d?

Reply:
Foi um exemplo que vi na internet. Mas já tô pensando em outra forma, pois vi na documentação que não deve ser usada.